server/views/layout.njk

```diff
@@ -121,8 +121,13 @@
 
 {% block bodyEnd %}
   {# Run JavaScript at end of the <body>, to avoid blocking the initial render. #}
+  <!-- Govuk Lib -->
   <script type="module" src="/assets/govukFrontendInit.js"></script>
+  <!-- MoJ Lib -->
   <script src="/assets/moj/all.js"></script>
-  <script src="/assets/mojFrontendInit.js"></script>
+  <script type="module" src="/assets/mojInit.mjs"></script>
+
+  <!-- DPR Lib -->
+  <script type="module" src="/assets/dpr/js/all.mjs"></script>
```

Comment:
I guess this only needs to included in pages that have reports too..

Reply:
Removed this from layout - thank you